embed_fixer/utils/download_media.py

```
@@ -20,12 +26,97 @@ def __init__(
+                        movflags="faststart",
+                        # libx264 + yuv420p requires even dimensions; round down to nearest even.
+                        vf="scale=trunc(iw/2)*2:trunc(ih/2)*2",
+                    )
+                    .global_args("-an")
+                    .overwrite_output()
+                    .run(quiet=True)
+                )
+            except ffmpeg.Error as e:
+                logger.error(f"ffmpeg failed to convert ugoira: {e.stderr}")
+                return None
+            except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)